### PR TITLE
Python 3.4/5/6 and PyPy support

### PR DESCRIPTION
**Pull request by jamadden**
**Needs a zope.app.tree release to finish**

- 100% test coverage
- Add tox.ini and .travis.yml and enable coveralls
- Rename .txt -> .rst
- Break test deps on zope.app.zcmlfiles and zope.app.renderer (Do we
  really actually need that flexibility at runtime too?)

### CODE REVIEW

**Comment by mgedmin** [comment]
You have PyPI rights now.

Also, is this still a WIP?  I'm reluctant to review a WIP ;)

**Comment by jamadden** [comment]
zope.app.tree just got released, so it's not a WIP anymore. I'm prepping the commit that removes the git install.

File: src/zope/app/preference/tests.py [hunk 1]
```diff
@@ -13,13 +13,151 @@
...
 """Tests for the Preferences System
 
-$Id$
 """
 import doctest
 import zope.app.preference.testing
 import zope.component.testing
 import zope.testing.module
+import importlib
+import unittest
```
Comment (mgedmin) [inline src/zope/app/preference/tests.py]: Move stdlib imports above zope.* imports?

**Review by mgedmin — APPROVED** [review]
LGTM

**Comment by jamadden** [comment]
Thank you!

**Pull request merged**